lib/cisco_node_utils/interface_hsrp_group.rb

```diff
@@ -393,6 +474,10 @@ def priority_level_set(pri, lower, upper)
       config_set('interface_hsrp_group', 'priority_level', @set_args)
     end
 
+    # This CLI can take forms like:
```

Comment:
Overall changes looking better but I think this one could also use keyword regexp instead of positional array.

https://github.com/cisco/cisco-network-node-utils/blob/b1c5459c699ae88e2b2ddf9e2e8866f6d062a0b2/lib/cisco_node_utils/interface_hsrp_group.rb#L481